static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -92,7 +92,7 @@ const TraceIssueLinkContainer = styled(Link)`
   font-size: ${p => p.theme.fontSizeMedium};
 
   &:hover {
-    background-color: ${p => p.theme.surface200};
+    background-color: ${p => p.theme.backgroundTertiary};
```

Comment:
`surface200` is so similar you can't even tell that the background color has changed.